charts/linkerd2/values.yaml

```diff
@@ -149,6 +149,13 @@ prometheusRuleConfigMapMounts: []
   # - name: recording-rules
   #   subPath: recording_rules.yml
   #   configMap: linkerd-prometheus-rules
+prometheusPersistence:
+  ### WARNING: prometheusPersistence is experimental and has not been tested/vetted by the Linkerd team.
```

Comment:
Also, As we are not creating a PersistentVolume, but just a persistentVolumeClaim expecting that there is a persistentVolume with the same storage class, and enough storage requirements, and access methods. A small comment for the same could be useful for users.

Reply:
added a note about pvc! thanks.